src/plugins.js

```diff
@@ -0,0 +1,131 @@
+export default {
+  // es2015
+  "transform-es2015-arrow-functions": {
+    edge: 13,
+    firefox: 45,
+    chrome: 49,
+    opera: 36,
+  },
+  "transform-es2015-block-scoped-functions": {
+    edge: 11,
+    firefox: 46,
+    chrome: 41,
+    opera: 28,
+  },
+  "transform-es2015-block-scoping": {
+    edge: 11,
+    firefox: 36,
+    chrome: 49,
+    opera: 36,
+  },
+  "transform-es2015-classes": {
+    edge: 13,
+    firefox: 45,
+    chrome: 49,
+    opera: 36,
+  },
+  "transform-es2015-computed-properties": {
+    edge: 12,
+    firefox: 34,
+    chrome: 44,
+    opera: 31
+  },
+  "check-es2015-constants": {
+    edge: 11,
+    firefox: 36,
+    chrome: 49,
+    opera: 36,
+  },
+  "transform-es2015-destructuring": {
+    edge: 14,
+    firefox: Number.NEGATIVE_INFINITY,
+    chrome: 52,
+    opera: 39,
+  },
+  "transform-es2015-for-of": {
+    edge: Number.NEGATIVE_INFINITY,
+    firefox: Number.NEGATIVE_INFINITY,
```

Comment:
@baer My other comments were mostly from experience, though this one is guessing :)

I just know that `for of` has to work with any iterable (i.e. anything with a `Symbol.iterator`), and I wouldn't expect very old firefox would have an implementation supporting that, even if it works with the basic case of Array.